src/inference/inference.ts

```diff
@@ -47,4 +57,136 @@ export class Inference {
     const response = await this._inferenceApi.embed(typedRequest);
     return new EmbeddingsList(response.model, response.data, response.usage);
   }
+
+  /** Rerank documents against a query with a reranking model. Each document is ranked in descending relevance order
+   *  against the query provided.
+   *
+   *  Note: by default, the ['text'] field of each document is used for ranking; you can overwrite this default
+   *  behavior by passing an {@link RerankOptions} `options` object specifying 1+ other fields.
+   *
+   *  @example
+   *  ```typescript
+   *  import { Pinecone } from '@pinecone-database/pinecone';
+   *  const pc = new Pinecone();
```

Comment:
nit: Can we add a line break here just to visually break this up a bit?

Reply:
yes!